Library/Formula/syncthing-inotify.rb

```diff
@@ -0,0 +1,59 @@
+require "language/go"
+
+class SyncthingInotify < Formula
+  desc "File watcher intended for use with Syncthing"
+  homepage "https://github.com/syncthing/syncthing-inotify"
+  url "https://github.com/syncthing/syncthing-inotify/archive/v0.6.5.tar.gz"
+  sha256 "430297896bb05396268fd29cc555eba6542b42263489784c9843f4daf625ac5c"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/cenkalti/backoff" do
+    url "https://github.com/cenkalti/backoff.git",
+        :revision => "6c45d6bc1e78d94431dff8fc28a99f20bafa355a" # not sure !
```

Comment:
Oh, this is unrelated to the `GitHub fork (not canonical repository)` error.  You're right that we get that error because this is a fork (using a fork is a good, but not perfect, indicator of a non-official repository).

My comment here was just about your `not sure!` comment.  If you're not sure what revision to use, just grab the most recent.